### PR TITLE
Add account schema endpoint on directories

### PR DESCRIPTION
**Pull request by cilim**
PR for issue https://github.com/stormpath/stormpath-sdk-ruby/issues/140
Please review.

### CODE REVIEW

File: lib/stormpath-sdk/resource/account_schema.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#
+# Copyright 2013 Stormpath, Inc.
```
Comment (DamirSvrtan) [inline lib/stormpath-sdk/resource/account_schema.rb]: 2013? :)

File: lib/stormpath-sdk/resource/account_schema.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+class Stormpath::Resource::AccountSchema < Stormpath::Resource::Instance
+  prop_reader :created_at, :modified_at
+
+  has_many :fields
```
Comment (DamirSvrtan) [inline lib/stormpath-sdk/resource/account_schema.rb]: It should also belong to a directory

File: lib/stormpath-sdk.rb [hunk 3]
```diff
@@ -60,6 +60,8 @@ module Resource
     autoload :PasswordPolicy, 'stormpath-sdk/resource/password_policy'
     autoload :EmailTemplate, 'stormpath-sdk/resource/email_template'
     autoload :PasswordStrength, 'stormpath-sdk/resource/password_strength'
+    autoload :AccountSchema, 'stormpath-sdk/resource/account_schema'
```
Comment (DamirSvrtan) [inline lib/stormpath-sdk.rb]: This should probably be called Schema, not Account Schema. The Account Schema should be the name of the association.

**Comment by cilim** [comment]
Comments fixed @DamirSvrtan!

File: lib/stormpath-sdk/resource/directory.rb [hunk 4]
```diff
@@ -28,6 +28,7 @@ class Stormpath::Resource::Directory < Stormpath::Resource::Instance
   has_one :custom_data
   has_one :password_policy
   has_one :account_creation_policy
+  has_one :account_schema, class_name: :schema, href: '/accountSchema'
```
Comment (DamirSvrtan) [inline lib/stormpath-sdk/resource/directory.rb]: By looking at the account_creation_policy.rb file and the relationship with the verification_email_templates, I'd say that you don't need this href.

Comment (cilim) [inline lib/stormpath-sdk/resource/directory.rb]: Yup, I accidentally left that while I was testing something. href: removed.

**Comment by DamirSvrtan** [comment]
@cilim I will merge it once the build on Travis passes.

**Comment by cilim** [comment]
@DamirSvrtan they passing :)

**Pull request merged**